CONTRIBUTING.md

```diff
@@ -99,6 +99,13 @@ All Alacritty changes are automatically verified by CI to conform to its rustfmt
 build is failing because of formatting issues, you can install rustfmt using `rustup component add
 rustfmt` and then format all code using `cargo fmt`.
 
+Unless otherwise specified, Alacritty follows the Rust compiler's style guidelines:
+
+https://github.com/rust-lang/api-guidelines
```

Comment:
I think it would be better to link directly to the book: https://rust-lang.github.io/api-guidelines/

That way, the guidelines are front and foremost (or at least, moreso than before), instead of requiring one to follow another link, or click through the various files in the repo.